### ISSUE TITLE
Player death will cause every player to drop their inventory seperately

### ISSUE THREAD

**Issue by EnchantedHay** [Matistan/MinecraftSharedInventory#3]
SharedHealth + SharedInventory

When players are sharing health, the death of one player will cause all players to die and drop their inventory separately, resulting in a bug of item duplication. 

Same bug may occur when players die simultaneously？

Any solution QAQ?

**Comment by Matistan** [comment]
Yes, there is a bug, where when some people's inventories change at the same exact game tick, some items might be gone, or might duplicate. This is because the plugin checks for inventory updates every single game tick, and can't check it faster, so I don't think there's a solution to that. I will see what I can do, to prevent this bug from happening when using both shared health and shared inventory though, since as you wrote, it is currently unplayable.

**Comment by Matistan** [comment]
This should be fixed in a newest update
